### PR TITLE
Videos

### PR DESCRIPTION
**Pull request by tamigoodall**

### CODE REVIEW

File: articles/videos/get-started/05_02-authorize-get-validate-id-tokens.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+description: How to get and validate ID Tokens before storing and using them.
+classes: video-page
+---
+# Authorize: ID Tokens and Access Control
```
Comment (rkhoriander) [inline articles/videos/get-started/05_02-authorize-get-validate-id-tokens.md]: Change this to match the section heading in the left nav?

File: articles/videos/get-started/08-brand-emails-error-pages.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
+---
+description: How to use email templates and customize error pages. 
+classes: video-page
+---
+# Brand: Emails and Error Pages
+
+Learn how to use Auth0 email templates and make changes to the reply email address, subject, redirect URL, and URL lifetime. You can configure your email provider and how verify emails look. Learn how to use your own error pages.
```
Comment (rkhoriander) [inline articles/videos/get-started/08-brand-emails-error-pages.md]: "how verify emails look" seems awkward. Maybe "customize how verify emails look"?

File: articles/videos/get-started/07_01-brand-how-it-works.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
+---
+description: Why your branding is important for your users and how it works with Auth0.
+classes: video-page
+---
+# Brand: How It Works
+
+Learn how branding works with Auth0 and how to add your company name and logo as well as choose colors for your pages.
```
Comment (rkhoriander) [inline articles/videos/get-started/07_01-brand-how-it-works.md]: Awkward. Maybe "Learn how branding works with Auth0. Add your company name and logo, and choose colors for your pages."?

File: articles/videos/get-started/05_01-authorize-id-tokens-access-control.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+---
+description: What an ID Token is and how you can add custom claims to make access control decisions for your users.
+classes: video-page
+---
+# Authorize: ID Tokens and Access Control
+
+Learn about using Auth0 authorization via the use of ID Tokens and understand what an ID Token is and how to add custom claims to customize them and make access control decisions for your users.
```
Comment (rkhoriander) [inline articles/videos/get-started/05_01-authorize-id-tokens-access-control.md]: A lot packed into this sentence. Maybe "Learn about Auth0 authorization via the use of ID Tokens. Understand what an ID Token is, and learn how to add custom claims to customize tokens and make access control decisions for your users."

File: articles/videos/get-started/04_01-authenticate-how-it-works.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
+---
+description: How user authentication works and various ways to accomplish it with Auth0.
+classes: video-page
+---
+# Authenticate: How It Works
+
+Learn about the difference between authentication, authorization, and access control. Understand when and why you might use each type of authentication method: First factors, second factors, and multi-factor. Learn about the OpenID Connect authentication protocol.
```
Comment (rkhoriander) [inline articles/videos/get-started/04_01-authenticate-how-it-works.md]: Don't capitalize "First". For consistency with other pages,  add "(OIDC)" after "OpenID Connect".

File: articles/videos/get-started/03-provision-import-users.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
+---
+description: How to move existing users to an Auth0 user store using automatic migration, bulk migration, or both.
+classes: video-page
+---
+# Provision: Import Users
+
+Learn how Auth0 allows you to move your existing users to an Auth0 user store and why you may want to use a combination of both automatic and bulk migration methods. Learn why its a good idea to have an API between the cloud and your legacy database and how custom database scripts work.
```
Comment (rkhoriander) [inline articles/videos/get-started/03-provision-import-users.md]: it's

File: articles/videos/get-started/02-provision-user-stores.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
+---
+description: How user profiles are provisioned within an Auth0 tenant.
+classes: video-page
+---
+# Provision: User Stores
+
+Learn how user profiles are provisioned within an Auth0 tenant. Users can come from a multitude of places: directly through creating an account using self sign up, indirectly through using their favorite trusted social media platform, or collectively through a trusted enterprise directory. Auth0 makes it simple to quickly connect your tenant to multiple types of user stores. We group these users stores into three categories: database, social, and enterprise.
```
Comment (rkhoriander) [inline articles/videos/get-started/02-provision-user-stores.md]: sign-up

Comment (rkhoriander) [inline articles/videos/get-started/02-provision-user-stores.md]: We always seem to leave off legal identity as a user store. Maybe this doesn't matter, but feels like it's something missing, particularly for European users.

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
+---
+description: What an Auth0 tenant is and how to configure it in the Auth0 Dashboard.
+classes: video-page
+---
+# Architecture: Your Tenant
+
+Learn what an Auth0 tenant is and how to configure it in the Auth0 Dashboard. Understand why you may want more than one tenant if you have different user communities, and also how you can use more than one tenant to support your Software Development Life Cycle. Learn the importance of tenant naming, use of custom domains, provisioning additional tenant administrators, and how to associate tenants with your Auth0 account.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: For consistency with other pages, add "(SDLC)" after "Software Development Life Cycle".

Last sentence is awkward. Missing verbs or an "and" somewhere?

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
+---
+description: What an Auth0 tenant is and how to configure it in the Auth0 Dashboard.
+classes: video-page
+---
+# Architecture: Your Tenant
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: Small quibble, but (almost) all the other sections begin with a verb. This one begins with a nominalization. Aren't they actually "architecting" in this step when choosing how to use their tenants? So wouldn't "Architect" be the proper word?

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+<div class="video-wrapper" data-video="ww3ykoy3vm"></div>
+
+## Transcript
+  
+<details>
+  <summary>Introduction</summary>
+  
+  In this video, we will talk about what an Auth0 tenant is, and show you how to configure it in the Auth0 Dashboard. We will also briefly explain why you may want to use more than one tenant if you have different user communities, and also how you can use more than one tenant in support of your Software Development Life Cycle. We’ll also describe the importance of tenant naming, use of custom domains, provisioning additional tenant admins, and how to associate tenants with your Auth0 account. 
+
+  Once you create your account in Auth0 you will be asked to create a Tenant. This is a logical isolation unit. The term tenant is borrowed from the phrase "software multi-tenancy" and refers to an architecture where a single instance of the software serves multiple tenants. No tenant can access the instance of another tenant, even though the software might be running on the same machine (hence the logical isolation).
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: "Once you create your account in Auth0,"

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+
+  When it comes to figuring out how to define your Auth0 tenants and accounts as part of your application integration, the value of investing time in the architectural “landscape” up front will pay dividends in the long run, and there are a number of things you’ll want to consider. 
+
+  It’s important to understand how your applications need to function within your infrastructure, and this will help you understand how to configure your tenants to accomplish your goals. How your Auth0 tenants are configured -- the architecture of your Auth0 deployment -- will form the basis for the grouping of your Auth0 assets to leverage features such as Single Sign On, centralized user profile management, and consolidated billing capabilities.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: I believe we use the `&mdash;` HTML code for em dashes.

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+
+  So you will need to decide how many different production tenants you will require. Most companies only need a single production tenant, but if you have a set of applications for just employees, you may want a separate tenant for that. In some situations, you may also need different production tenants for different customer user communities. 
+
+  Another reason you would want to create multiple tenants is to support your Software Development Life Cycle. Auth0 can fit into your process by allowing you to have a separate tenant for each phase - such as one for development, one for testing, and one for production. Even if you don’t use an SDLC methodology, you will most likely want to create at least two tenants: one for development and one for production.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: Should be an em dash here.

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+  So you will need to decide how many different production tenants you will require. Most companies only need a single production tenant, but if you have a set of applications for just employees, you may want a separate tenant for that. In some situations, you may also need different production tenants for different customer user communities. 
+
+  Another reason you would want to create multiple tenants is to support your Software Development Life Cycle. Auth0 can fit into your process by allowing you to have a separate tenant for each phase - such as one for development, one for testing, and one for production. Even if you don’t use an SDLC methodology, you will most likely want to create at least two tenants: one for development and one for production. 
+
+  You may want to name one tenant “company-dev” to serve as a shared environment where your development work occurs, and name another tenant “company-qa” for testing your Auth0 integration. You can then name a third tenant “company-prod”, to serve as your production tenant.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: No comma necessary after "company-prod".

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+
+  You may want to name one tenant “company-dev” to serve as a shared environment where your development work occurs, and name another tenant “company-qa” for testing your Auth0 integration. You can then name a third tenant “company-prod”, to serve as your production tenant. 
+
+  You can also create tenants to serve as sandboxes to test potential changes, like different deployment scripts, without compromising your environment. Auth0 lets you create as many free tenants as you like, but you may be limited for the number of tenants where all paid features are enabled. You can have up to three tenants where all features are available.
+
+  It’s also important to consider exactly what you will name your tenants. The ultimate goal is to end up with one or more production tenants that are branded exactly the way you want them to appear to your users. Tenant naming patterns are very important, so it’s good to plan the tenant names in advance because you won’t be able to change them once you create them,or use them again if you delete them.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: Missing space before "or".

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+
+  Out-of-the-box, client facing URLs are Auth0 branded, however, we recommend using the Auth0 custom domain capability to provide a consistent corporate identity and to also address potential user confidence concerns before they arise. Your company name as part of the URL, for example, will support your brand and should tell your users that they can trust that they are in the right place to enter their identity information. It’s also harder to phish your domain if you have a Custom Domain (a.k.a. vanity) URL because the phisher also has to create one to mimic yours. A centralized domain for authentication across multiple product or service brands so users see a consistent interface is also important. And keep in mind that some browsers make it harder to communicate in an iFrame if you don’t have a shared domain.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: "Out-of-the-box, client-facing URLs are Auth0 branded; however,"

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 16]
```diff
@@ -0,0 +1,159 @@
...
+
+  Having more than one Auth0 Dashboard administrator is a good idea, and periodically reviewing the list of Auth0 Dashboard administrators to see that the right people have access to your Auth0 tenants will help you make sure that each person has a legitimate need for admin access. You should make sure that former employees no longer have access. We also recommend that you enable multi-factor authentication for all your admins for added security. Having more than one admin can alleviate the burden of Auth0 tenant administration, but if you only have one admin they may get locked out if they lose their phone. if you have more than one admin, however, another can temporarily disable MFA for the admin who lost their phone.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: Capitalize "If" in last sentence.

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 17]
```diff
@@ -0,0 +1,159 @@
...
+</details>
+
+<details>
+  <summary>Advanced settings</summary>
+ 
+  There are also some advanced tenant settings that you can configure for your tenant. There are a couple of things you need to consider when you configure these items. 
+
+  Logout is the act of terminating an authenticated session. It is a security best practice to terminate sessions when they’re no longer needed to avoid a potential takeover by unauthorized parties. This is usually achieved via the provision of some “logout” option on the UI. There are multiple types of sessions that could be created when a user logs in. For example, a local application session, Auth0 session, and/or a third party Identity Provider session. You’ll need to determine which should be terminated when the user clicks on any “logout” option. You can configure the application to have a logout redirect URL.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: "third party" should be "third-party".

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 18]
```diff
@@ -0,0 +1,159 @@
...
+
+  Session timeout settings allow you to specify when the SSO cookie times out. The value you set is the login session lifetime which is how long the session will stay valid, measured in minutes. The default setting is set for 10,080 minutes or 7 days. 
+
+  Your contractual agreement with Auth0 should cover all the tenants you want to use. Make sure all your tenants are associated with your company account. If you have developers that want to create their own sandboxes for testing, make sure the tenants are associated with your account so they have the same permissions and Auth0 features available too.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: "developers that" should be "developers who" (although I guess this is a transcript, so if the person said "that", then I guess it stays?)

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 19]
```diff
@@ -0,0 +1,159 @@
...
+
+  Your contractual agreement with Auth0 should cover all the tenants you want to use. Make sure all your tenants are associated with your company account. If you have developers that want to create their own sandboxes for testing, make sure the tenants are associated with your account so they have the same permissions and Auth0 features available too. 
+
+  If you need to add a tenant to your account, contact the Auth0 Support Center at https://support.auth0.com. Specify your production tenant so you can get higher rate limits than non-production tenants. Only one tenant per subscription can be set as the production tenant. 
+</details>
+
+<details>
+  <summary>Summary</summary>
+  
+  So, to summarize, we described what an Auth0 tenant is, how to configure it, and why you may need more than one tenant. We  also described the importance of tenant naming, and how to associate tenants with your Auth0 accounts to save money and leverage Auth0 features across your organization.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: Extra space after "We" in second sentence.

File: articles/videos/get-started/01-architecture-your-tenant.md [hunk 20]
```diff
@@ -0,0 +1,159 @@
...
+
+  In the next video, we’ll talk about provisioning your users and configuring user stores, and in a future video we’ll talk more about Custom Domains too.
```
Comment (rkhoriander) [inline articles/videos/get-started/01-architecture-your-tenant.md]: Add comma after "in a future video".

File: articles/videos/get-started/01-architecture-your-tenant.md (hunk 9 above)
Comment (tamigoodall) [reply articles/videos/get-started/01-architecture-your-tenant.md]: Yes - much better.

File: articles/videos/get-started/01-architecture-your-tenant.md (hunk 8 above)
Comment (tamigoodall) [reply articles/videos/get-started/01-architecture-your-tenant.md]: Fixed

File: articles/videos/get-started/02-provision-user-stores.md (hunk 7 above)
Comment (tamigoodall) [reply articles/videos/get-started/02-provision-user-stores.md]: I'll add this to the list for the next revision.

File: articles/videos/get-started/05_01-authorize-id-tokens-access-control.md (hunk 4 above)
Comment (tamigoodall) [reply articles/videos/get-started/05_01-authorize-id-tokens-access-control.md]: Much better

**Review by rkhoriander — DISMISSED** [review]

**Review by rkhoriander — APPROVED** [review]

**Pull request merged**